container/import.bzl

```diff
@@ -102,7 +113,7 @@ def _container_import_impl(ctx):
 container_import = rule(
     attrs = {
         "config": attr.label(allow_files = [".json"]),
-        "layers": attr.label_list(allow_files = tgz_filetype),
+        "layers": attr.label_list(allow_files = tar_filetype),
```

Comment:
I think this should be `[tar_filetype, tgz_filetype]`

Reply:
Done.